npm/windows.Dockerfile

```diff
@@ -8,9 +8,7 @@ COPY . .
 RUN GOOS=windows CGO_ENABLED=0 go build -v -o /usr/local/bin/azure-npm.exe -ldflags "-X main.version="$VERSION" -X "$NPM_AI_PATH"="$NPM_AI_ID"" -gcflags="-dwarflocationlists=true" npm/cmd/*.go
 
 # intermediate for win-ltsc2022
-FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as ltsc2022
-
-FROM ${OS_VERSION} as windows
+FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as windows
```

Comment:
Are we not able to use `windows-host-process-containers-base-image` here?

Reply:
I don't know and I'm not trying to make big changes to NPM, so I made the build work and left it

Reply:
Fair enough. We can look at this another time.

Reply:
@huntergregory FYI since this needs an NPM reviewer - there should be no net change to NPM here, I only had to update the Dockerfile to match the pattern of everything else in the build matrix